packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -21,8 +21,8 @@ export const OAUTH_REALTIME_CHANNEL = 'oauth-popup'
 export const checkOAuthData = (konnector, data) => {
   if (!data) return false
 
-  const { key, oAuthStateKey } = data
-  if (!key || !oAuthStateKey) return false
+  const { oAuthStateKey } = data
+  if (!oAuthStateKey) return false
```

Comment:
Why this removal of the `key` attribute? Can you do it in another commit and explain why?

Reply:
can you edit the JSDoc please? 

Also since we don't use the key anymore, maybe we can delete it? 

https://github.com/cozy/cozy-libs/blob/67902727394c9ef3400e32bc3cbddf5468ef332c/packages/cozy-harvest-lib/src/components/OAuthWindow.jsx#L92 
and so here https://github.com/cozy/cozy-libs/blob/67902727394c9ef3400e32bc3cbddf5468ef332c/packages/cozy-harvest-lib/src/components/OAuthWindow.jsx#L87 

But this is weird since we still need it here https://github.com/cozy/cozy-libs/blob/67902727394c9ef3400e32bc3cbddf5468ef332c/packages/cozy-harvest-lib/src/components/OAuthWindow.jsx#L97 ? So why not just changing the method `checkOauthData` to only take (konnector, oAuthStateKey) ? (aka removing the need to pass the all object?)

Reply:
The key is still needed in case of OAuth danse success, to allow OAuthForm to get the accountId, to be able to update it and also create a Trigger related to it.